CHANGELOG.md

```diff
@@ -4,10 +4,14 @@ Change Log
 2.1.0
 -----
 
+* Added filtering to the groups and permissions selection boxes on the invitation page.
+* Added check for the usernames and emails on the invitation page to show whether they're in use before submitting the form.
 * Added setting to show emails on the accounts page to superusers and users who can create invitations.
 * Added setting to limit the timeframe for showing open invitations and registrations on the accounts page.
-* Made some small UI tweaks on the invitations and accounts pages.
+* Tweaked the UI on the invitations and accounts pages.
 * Fixed remaining sorting difference that occurred after resending invitation email.
+* Fixed some other small UI errors, particularly on the invitations and accounts pages.
+* Changed the dev environment to output emails directly stdout.
```

Comment:
Do you want this to say "directly _to_ stdout"?

Reply:
ha, yep, I _did_ want it to say that